### ISSUE TITLE
Encoding issue?

### ISSUE THREAD

**Issue by cryoguy** [wasamasa/nov.el#59]
Did anybody encounter this issue?   Looks like it fails to encode apostrophe and space etc. I am using Arch Linux with Emacs 25.3.
-----------------------------------
I didn鈥檛 find the last line quite satisfying, so for several weeks I reflected on it, questioned it, wrestled with it, until one day the answer I was looking for came to me: 鈥淚f you want to be happy for a lifetime, give yourself to others.鈥

**Comment by wasamasa** [comment]
I don't really understand this one, did you copy-paste rendered text from an EPUB file? If yes, then please answer the following questions:

- Can you reproduce the issue with a minimal Emacs configuration from `emacs -Q`?
- Is the EPUB file considered valid by `epubcheck`?
- Does it display correctly in other EPUB readers?

If you can answer all of these with yes, please send me a copy of the EPUB file.

**Comment by cryoguy** [comment]
Thanks for your prompt reply!  I copied it after encoded by nov.el, and the followings are what I found in responding to your guidance:

- The epub file displayed ok with minimal Emacs configuration
- The epub file was validated by epubcheck with no glitches and errors
- The file display fine in other epub reader such as calibre

I guess the misbehavior is resulting from a incompatibility with other packages.  I will look into it when I have a chance and report it back to you.

Thanks!
